assists/subsystem.py

```diff
@@ -103,7 +103,7 @@ def firewall_expand( tree, subnode, verbose = 0 ):
 
         # the first item is "block" (0) and the second is the priority (default 0), so
         # we use <0 0>
-        firewall_prop = LopperProp( "firewallconfig-default", -1, subnode.parent, [ 0, 0 ] )
+        firewall_prop = LopperProp( "firewallconf-default", -1, subnode.parent, [ 0, 0 ] )
```

Comment:
@its-izhar  can you also update CDO plugin with 1 line fix at https://github.com/devicetree-org/lopper/blob/master/assists/cdo.py#L213 in your next PR for the plugin? Otherwise I can do so

Reply:
Yes, I'm already taking care of this in my upcoming pull request. you don't need to do it. thanks!